etna/insights/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.1.8 on 2021-07-08 13:26
```

Comment:
Could you squash these migrations down to a single file:

Roll back the insights migration:

`python manage.py migrate insights zero`

Delete the existing migration files

Recreate:
`python manage.py makemigrations insights`

Reply:
and done.